File: Bachelor/Semester4/Systems_for_Design_and_Implementation/C#/ProiectCShMPP/Networking/RpcProtocol/Request.cs
using System;

namespace Networking.RpcProtocol
{
    [Serializable]
    public class Request
    {
        public RequestType Type
        {
            get;
            set;
        }

        public Object Data
        {
            get;
            set;
        }

        public override string ToString()
        {
            return Type+" "+Data;
        }
    }
}

File: Bachelor/Semester5/Parallel_and_Distributed_Programming/P1/Networking/src/main/java/rpcprotocol/Request.java
package rpcprotocol;

import java.io.Serializable;

public class Request implements Serializable {
    private RequestType type;
    private Object data;

    public Request(RequestType type) {
        this.type = type;
    }

    public Request(RequestType type, Object data) {
        this.type = type;
        this.data = data;
    }

    public RequestType getType() {
        return type;
    }

    public void setType(RequestType type) {
        this.type = type;
    }

    public Object getData() {
        return data;
    }

    public void setData(Object data) {
        this.data = data;
    }

    @Override
    public String toString() {
        return "Request{" + "type=" + type +
                ", data=" + data +
                '}';
    }
}